config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'items#index'
```

Comment:
is this how we write our routes?

Reply:
So I am not sure how to fix this if I change it the tests all fail.